packages/blade/src/components/Alert/Alert.stories.tsx

```diff
@@ -35,10 +36,48 @@ const Page = (): ReactElement => {
         explanations inside the system in a prominent way.
       </Subtitle>
       <Title>Usage</Title>
-      <Link withArrow={true} href={figmaURL} target="_blank" rel="noreferrer noopener">
-        View in Figma
-      </Link>
-      <Highlight language="tsx">{`import { Alert } from '@razorpay/blade/components' \nimport type { AlertProps } from '@razorpay/blade/components'`}</Highlight>
+      <FigmaEmbed title="Alert Figma Designs" src={figmaURL} />
```

Comment:
The figma embed should come above the `<Title>Usage</Title>`?

Reply:
I feel yes cause figma doesn't mention how to use the component in itself. It's more of an overview of possible variants and types of component right now.

Reply:
i think you forgot to make this change in just this file. other places look fine over here it looks like this
<img width="1089" alt="Screenshot 2022-11-07 at 1 27 21 PM" src="https://user-images.githubusercontent.com/11384858/200255961-7139927f-d281-4835-a5ed-ee42c02f0800.png">

Reply:
oops yes missed this